pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+		xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>ep6-TH-Bingen</groupId>
+	<artifactId>project_alpha_omega</artifactId>
+	<version>1.0</version>
```

Comment:
During development, you should add `-SNAPSHOT` here. ([Why?](https://stackoverflow.com/a/5901460/2525313))